benchto-driver/src/main/java/io/trino/benchto/driver/listeners/BenchmarkServiceExecutionListener.java

```diff
@@ -167,6 +171,38 @@ private void checkClocksSync()
                 });
     }
 
+    @Override
+    public Future<?> concurrencyTestExecutionFinished(List<QueryExecutionResult> executions)
```

Comment:
do you know, if benchto-server UI still works after these changes and reported benchmarks?

Reply:
It displays an execution, but doesn't show any details / metrics. I can check what is missing and if it's easy to get some proper output displayed in UI.